### PR TITLE
fix(Cypress): add chrome to travis

### PR DESCRIPTION
**Pull request by AsToNlele**
Unset chrome as default browser in CI

## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [ ] General Coding Practices

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -147,7 +147,7 @@
     "server:ctr": "node src/server/generateServerKey.js",
     "start": "fec dev",
     "start:proxy": "PROXY=true fec dev",
-    "travis:verify": "npm-run-all lint test && ./codecov -cF jest && npm run test:ct -- --browser chrome && ./codecov -cF cypress && npm run build",
+    "travis:verify": "npm-run-all lint test && ./codecov -cF jest && npm run test:ct && ./codecov -cF cypress && npm run build",
```
Comment (mkholjuraev) [inline package.json:150]: This looks more complex than needed. I would suggest removing this script and making travis.yml same as in inventory app: https://github.com/RedHatInsights/insights-inventory-frontend/blob/master/.travis.yml

**Comment by AsToNlele** [comment]
I've ran cypress separately and it succeeded, I'm trying to put the build into separate jobs if it helps

**Comment by AsToNlele** [comment]
Putting cypress separate didn't fix the issue. Downgraded cypress to the version we use in inventory, let's see

**Comment by AsToNlele** [comment]
let's move on with adding chrome as an addon, couldn't figure out why electron sometimes doesn't even boot.

**Comment by mkholjuraev** [comment]
@gkarat is this not one of the identified flakiness issues during your investigation?

**Review by mkholjuraev — APPROVED** [review]
After doing some research, I believe this is safe enough and usual stuff. LGTM from my side.

**Pull request merged**